features/steps/symfony-steps.rb

```diff
@@ -0,0 +1,45 @@
+require_relative "../lib/symfony"
+
+Given("I start the symfony fixture") do
+  steps %{
+    Given I start the service "#{Symfony.fixture}"
+    And I wait for the host "localhost" to open port "#{Symfony.fixture_port}"
+  }
+end
+
+When("I navigate to the route {string}") do |route|
+  Symfony.navigate_to(route)
+end
+
+Then("the event {string} matches the current PHP version") do |path|
+  steps %{
+    Then the event '#{path}' starts with '#{ENV["PHP_VERSION"]}'
+    And the event '#{path}' matches '^(\\d\\.){2}\\d+$'
+  }
+end
+
+Then("the event {string} matches the current major Symfony version") do |path|
+  steps %{
+    Then the event '#{path}' starts with '#{ENV["SYMFONY_VERSION"]}'
+    And the event '#{path}' matches '^(\\d\\.){2}\\d+$'
+  }
+end
+
+Then("the Symfony response matches {string}") do |regex|
+  wait = Maze::Wait.new(timeout: 10)
+  success = wait.until { Symfony.last_response != nil }
+
+  raise 'No response from the Symfony fixture!' unless success
+
+  assert_match(Regexp.new(regex), Symfony.last_response)
+end
+
+# TODO this should go in Maze Runner itself
```

Comment:
Let's just get it added now.  Send a ticket my way and I'll turn it around same day (if at all possible).

Reply:
(Update: I've added it myself now and put the review on you).

Reply:
Thanks, forgot about this!